functionalDB/chapter.md

```diff
@@ -1,103 +1,104 @@
-# Designing a database like an archaeologist
+<!-- American spelling --> 
+# CircleDB: An Archaeology-Inspired Database

```

Comment:
I'm fine with starting the title with the name. I would still prefer to keep it "Designing a database like an archaeologist", as it emphasises the main points here, which are the design and the usage of different perspective, as oppose to taking ideas from the domain of archaeology